install.sh

```diff
@@ -316,7 +319,10 @@ setup_env() {
         fi
         BINARY_SOURCE=local
     else
-        BINARY_SOURCE=remote
+        if [ -z "${CATTLE_AGENT_BINARY_LOCAL_LOCATION}" ]; then
+            BINARY_SOURCE=remote
+        fi
+
```

Comment:
If `CATTLE_AGENT_BINARY_LOCAL` is not `true`, but `CATTLE_AGENT_BINARY_LOCAL_LOCATION` is set, then `BINARY_SOURCE` will be unset. Is this the desired outcome?

Reply:
I believe so. Admittedly I was confused by this initially, but this was what already existed.

Reply:
It looks like you are modifying what is existing.

Reply:
Fixed, not sure what I was thinking there